### PR TITLE
update the sort docs

### PR DESCRIPTION
**Pull request by osman**
- also fix the failing tests where 'type:"Field"' is missing

### CODE REVIEW

File: src/main/java/com/groupbyinc/api/Query.java [hunk 1]
```diff
@@ -1117,7 +1117,11 @@ public Query setRestrictNavigation(String name, int count) {
    * relevance. It is possible to specify multiple sort criteria. The criteria order matters, as the records will be
    * sorted by the first criteria and then any matches will be tie-broken using the next criteria. Given an example
    * where the sort is specified as `category` then `_relevance`, results will be sorted first by `category` and
-   * relevance will only affect the order between records that have the same category.
+   * relevance will only affect the order between records that have the same category. Records can also be sorted by
+   * a specific ID as well when you want to return items in a specific order. If a record ID is included as a sort,
+   * but that record not a part of the resultset, that item will not be included (unlike push to top). There is a limit
```
Comment (olyas) [inline src/main/java/com/groupbyinc/api/Query.java]: "resultset" --> "result set" (or add to searchandiser.dic)

**Review by olyas — CHANGES_REQUESTED** [review]
@osman One typo

**Comment by olyas** [comment]
Hello world

**Comment by olyas** [comment]
I approve for the record

**Review by ajakop — APPROVED** [review]

**Pull request merged**